Plugins/NativeCamera/Source/UWP/NativeCameraImpl.cpp

```diff
@@ -14,7 +14,7 @@ namespace Babylon::Plugins
 
     arcana::task<Camera::Impl::CameraDimensions, std::exception_ptr> Camera::Impl::Open(uint32_t /*width*/, uint32_t /*height*/, bool /*frontCamera*/)
     {
-        throw std::runtime_error{ "HW Camera not implemented for this platform." };
+        return {};
```

Comment:
I'm assuming this comment matches this code?
> with win32/x86, a method that throw is discarded by the compiler and an 'optimized' code path detect a lambda to not be reachable (NativeVideo). It only happens with MSVC/x86

I don't get it. This is technically not correct?

Reply:
Yes! it's correct. code can be discarded and unreachable code warning should(may) happen. but it's only the case for x86 MSVC...other compiler do not optimize it...or do not have a warning in that case.

Reply:
I don't understand. This function is not implemented. We can't return an empty task. That's not correct behavior?

Reply:
Yes, not sure about this one. I prefer to revert the empty task change and disable UB for this project.